frontend/src/components/TransferDatabaseForm/ResetDatabaseLabel.vue

```diff
@@ -0,0 +1,247 @@
+<template>
```

Comment:
Should this be called Set or Update instead of Reset?

Reply:
Or Select?

Reply:
Yeah, Select sounds better.

Reply:
renamed to Select